src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,83 @@
+.clock {
+    position: relative;
+    @include clock-size(300px);
+
+    background-color: $main-clock-color;
+    border: 7px solid $secondary-color;
+    border-radius: 50%;
+
+    transform: perspective(75em);
+    box-shadow: rgba(22, 31, 39, 0.42) 0px 60px 123px -25px,
+        rgba(19, 26, 32, 0.08) 0px 35px 75px -35px;
+
+
+    &__hand {
+      position: absolute;
+      left: 50%;
+      left:0;
+      right:0;
+      margin:0 auto;
+
+      height: 150px;
+
+      border-radius: 8px;
+      transform-origin: bottom left;
+
+      animation-timing-function: linear;
+      animation-iteration-count: infinite;
+
+      &--hour {
+        height: 100px;
+        width: 10px;
+        z-index: 100;
```

Comment:
Mentor told us to don't put a such high z-index property, try to use smaller one

Reply:
changed it